agent/tests/test_fw_support.c

```diff
@@ -13,8 +13,15 @@ tlib_parallel_info_t parallel_info
 
 static void test_fw_supportability_metrics(void) {
 #define LIBRARY_NAME "php-package"
+#define LIBRARY_MAJOR_VERSION "7"
+#define LIBRARY_MAJOR_VERSION_2 "10"
+#define LIBRARY_MAJOR_VERSION_3 "100"
+#define LIBRARY_MAJOR_VERSION_4 "1.23"
+#define LIBRARY_MAJOR_VERSION_5 "12.34"
+#define LIBRARY_MAJOR_VERSION_6 "123.45"
```

Comment:
Nice test additions!

maybe something starting with a zero too?
like `0.4.5`?

Reply:
Good idea, I added more tests: [97c639e](https://github.com/newrelic/newrelic-php-agent/pull/868/commits/97c639e0e64aeb8e5e4a6d4db7c25de4aa928c0e). Instead of doing `NRINI(force_framework) = (nrframework_t)LIBRARY_NAME;`, I instead did `NRINI(force_framework) = true;` which allows us to test the forced code path without having to typecast `LIBRARY_NAME` since it doesn't exist in `nrframework_t`.